project/AppDependencies.scala

```diff
@@ -3,12 +3,15 @@ import sbt._
 
 object AppDependencies {
 
+  val bootstrapVersion = "7.19.0"
+
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % "5.24.0",
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % bootstrapVersion,
     "com.typesafe.play"       %% "play-json-joda"             % "2.9.2"
   )
 
   val test: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"  % bootstrapVersion,
```

Comment:
Not sure but might be worth adding `% "test"` here, might fix your platops comment:
```
uk.gov.hmrc.bootstrap-test-play-28 in [project/AppDependencies.scala](https://github.com/hmrc/trusts-auth/blob/DDCE-4029/project/AppDependencies.scala#L14) is not limited to test scope but pulls in test dependencies
```
although I am not sure 🤷